tripy/tripy/frontend/module/sequential.py

```diff
@@ -0,0 +1,185 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Union, Tuple, Dict, Iterator, Any
+from dataclasses import dataclass
+import copy
+
+from tripy import export
+from tripy.common.exception import raise_error
+from tripy.frontend.module import Module
+
+
+@export.public_api(
+    document_under="modules/sequential.rst",
+    autodoc_options=[
+        ":exclude-members: __getattr__",
```

Comment:
Could you double check that this is working?

Reply:
Actually, I will remove this, since this matters only when we have `special_members` ON.